packaging/source/buildpackage.sh

```diff
@@ -3,17 +3,19 @@
 
 set -o errexit -o pipefail || exit $?
 
-if [ $# -ne "2" ]; then
-	echo "Usage: $(basename "$0") tag outputdir"
-	exit 1
-fi
```

Comment:
I don't understand why this needs to be removed.

Reply:
The values of `$1` and `$2` are now both inferred if absent. Therefore supplying them is optional, and this block forces them to be included.

Reply:
The tag is required for every release. A bogus value could mean games desync so this is going into the wrong direction.